Makefile

```diff
@@ -64,8 +64,10 @@ install-zlib:
 	@rm -rf zlib-1.2.12
 	@rm -f zlib-1.2.12.tar.gz
 
-# "check" depends "build". out/libindexer.a seems needed to process cgo directives
-check: build check-ltag check-dco
+check: check-ltag check-dco check-lint
+
+# "check-lint" depends "build". out/libindexer.a seems needed to process cgo directives
+check-lint: build
```

Comment:
Instead of `build`, can we just use `pre-build`?

Reply:
Yeah, that works and saves 30s-1minute.